heroku/cli
# refactor: migrate members:set to oclif/core

[Work item](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001awBZlYAM/view)

## Testing
- Check out this branch and run `yarn` and `yarn build`
- Pick a team to test on (I like to use hk-front-end-online-test-team and hk-front-end-legacy-ent-team)
- Changing the role of a member on a team
    - The role can be set to admin, member, collaborator, or owner (hk-front-end-online-test-team only accepts admin or member)
    - Change the role using `./bin/run members:set -t TEAM-NAME -r ROLE EMAIL
